rules/do-you-create-a-migration-plan/rule.md

```diff
@@ -23,14 +23,16 @@ Migrating from .NET Framework (4.x) to the latest .NET (5+) brings huge advantag
 ## Preparation
 The migration to a newer version of .NET is the perfect opportunity for you to take stock of your current application architecture, and address any technical debt your app has accumulated. Trying to migrate an application that's poorly architected or carrying a lot of tech debt will make your migration **exponentially harder**. Therefore, now is the time to perform a full audit of your app and ensure you have PBIs to rectify these problems before you consider it "migration-ready".
 
-### Dependency analysis
+### Manual ependency analysis
```

Comment:
```suggestion
### Manual Dependency analysis
```

Reply:
updated, i will use lower case D for 'dependency'.